spec/dataframe_spec.rb

```diff
@@ -561,6 +570,13 @@
 
         expect(@df_mi).to eq(answer)
       end
+
+      it "assigns correct name given empty dataframe" do
+        df_empty = Daru::DataFrame.new([], index: @multi_index, order: @order_mi)
+        df_empty[:c, :one, :bar] = 1..12
+
+        expect(df_empty[:c, :one, :bar].name).to eq "conebar"
+      end
     end
```

Comment:
@v0dro I am wondering whether this test is any useful or not because the bug that was there had to deal with assigning correct name to column when DF didn't had any index built. In case of Multi Index, the DF cannot be created without first specifying the index, so the test which I had put up for Multi Index doesn't has anything to do with the bug. If I could built a Multi Index DF without assigning the index first, then I think the test would be useful.